src/components/ShowError/ShowError.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import cn from 'classnames';
+import { useEffect, useState } from 'react';
+
+type Props = {
+  error: string,
+};
+
+export const ShowError: React.FC<Props> = ({ error }) => {
+  const [isHidden, setIsHidden] = useState(true);
```

Comment:
you don't need this state if you render the component conditionally from the parent one